.travis.yml

```diff
@@ -31,34 +29,37 @@ script:
 
 jobs:
   include:
-    - php: 5.5
-      env: deps=low SYMFONY_DEPRECATIONS_HELPER=weak
+    # Tests the lowest set of depedencies
+    - php: 7.1
+      env: deps=low SYMFONY_DEPRECATIONS_HELPER=999999
       install:
         - COMPOSER_MEMORY_LIMIT=-1 travis_retry composer update -n --prefer-lowest --prefer-stable --prefer-dist
 
-    - php: 5.6
-      env: SYMFONY_VERSION="2.8.*"
-
-    - php: 7.0
-      env: SYMFONY_VERSION="3.2.*"
+    # Test against latest Symfony 3.4 stable
+    - php: 7.1
+      env: SYMFONY_VERSION="3.4.*" SYMFONY_DEPRECATIONS_HELPER=999999
 
+    # Test against latest Symfony 3.4 dev version
     - php: 7.1
-      env: stability=RC SYMFONY_DEPRECATIONS_HELPER=weak SYMFONY_VERSION="3.4.*"
+      env: stability=dev SYMFONY_DEPRECATIONS_HELPER=999999 SYMFONY_VERSION="3.4.*"
       install:
         - travis_retry composer update -n --prefer-dist
 
+    # Test against latest Symfony 4.1 dev version
     - php: 7.1
-      env: stability=dev SYMFONY_DEPRECATIONS_HELPER=weak SYMFONY_VERSION="4.1.*"
+      env: stability=dev SYMFONY_DEPRECATIONS_HELPER=999999 SYMFONY_VERSION="4.1.*"
       install:
         - composer require --dev "symfony/messenger:4.1.*" --no-update
         - travis_retry composer update -n --prefer-dist
 
+    # Test against latest Symfony 4.2 dev version
     - php: 7.1
-      env: stability=dev SYMFONY_DEPRECATIONS_HELPER=weak SYMFONY_VERSION="4.2.*"
+      env: stability=dev SYMFONY_DEPRECATIONS_HELPER=999999 SYMFONY_VERSION="4.2.*"
```

Comment:
Shouldn't that one fail on deprecations?
Running with the latest version of Symfony should always be deprecation.
That will help you spot where the bundle is behind and fix.

Reply:
This one just tests against an upcoming 4.2 release which should not contain any new deprecations anyways. We always test against the latest stable version of Symfony using the regular build jobs.

It may be worth adding a separate job that tests against the master branch of Symfony (similar to how we currently test with `'minimum-stability': 'alpha'`) to report upcoming deprecations, but then I'd also have that build allow failures so as to not break unrelated PRs. What do you think?

Reply:
> This one just tests against an upcoming 4.2 release which should not contain any new deprecations anyways

so better ensure it by failing if this is not verified? this would prevent merging a PR in the bundle that'd use a deprecated feature.

> adding a separate job that tests against the master branch

that'd make sense yes, failing on deprecations but in allowed failure. It would help identify which parts of the bundle need to be upgraded in preparation for the next version of Symfony.

Reply:
That sounds good. Will update the pipeline 👍

Reply:
Done!